### PR TITLE
feat: Adding runtime flag support

### PR DESCRIPTION
**Pull request by robtarr**
I had to update some dependencies, and this one got a little boated. :(

It's now only relying on Babel to created the client-side distributable file `dist/featurama-client.js`. Everything else is using ES6 features built into Node >= 4.3.1.

### CODE REVIEW

File: dist/featurama.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ module.exports = function (options) {
     };
 
     files.forEach(function (file) {
-      var data;
+      var data = void 0;
```
Comment (patricksimpson) [inline dist/featurama.js]: Had to look this up, :+1:

File: tasks/watch.js [hunk 2]
```diff
@@ -17,10 +19,10 @@ watch.createMonitor('spec', options, (monitor) => {
   });
 });
 
-watch.createMonitor('lib', options, (monitor) => {
+watch.createMonitor('src', options, (monitor) => {
   monitor.on('changed', (f, curr, prev) => {
     shell.exec('npm run compile');
     shell.exec('npm test');
-    shell.exec('npm run docs');
+    // shell.exec('npm run docs');
```
Comment (patricksimpson) [inline tasks/watch.js]: do you want to leave this in commented?

Comment (robtarr) [inline tasks/watch.js]: Yeah, that's just the watch command. It's fine to leave in, it's actively being worked on.

File: dist/featurama.js (hunk 1 above)
Comment (robtarr) [inline dist/featurama.js]: This the generated file. We shouldn't make changes in here. I did update the `src` file to do the assignment on the same line as the declaration.

File: tasks/watch.js (hunk 2 above)
Comment (patricksimpson) [inline tasks/watch.js]: Cool!

**Comment by patricksimpson** [comment]
Looks good here, merging :+1:

**Pull request closed without merging**